src/tv2-common/videoSwitchers/TriCaster.ts

```diff
@@ -38,6 +40,8 @@ const TRANSITION_MAP: Record<TransitionStyle, TSR.TriCasterTransitionEffect> = {
 	[TransitionStyle.STING]: 5 // not really supported??
 }
 
+const DVE_OVERLAY_INPUT_NUMBER: number = 5
```

Comment:
The type is a bit unnecessary here.

Reply:
That's a matter of preference.
Personally I think it's a good habit to have so you don't accidentally forget the type when writing a variable where the type is not self explanatory i.e: `const carIds = getCarIds()`. In the previous snippet, what type is `carIds`? Is it `string[]`, `number[]`, `CarId[]` or something fourth?

Reply:
But I can refactor if you want. (Can see I have several constant without the type specified)

Reply:
I like it, but it's rare to find in Sofie.

Reply:
Personally I think it is good to have for complex shapes, but unnecessary for primitive literals. I can understand having it for consistency and for "remembering" to add types to constants.